metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/ReferenceSequence.java

```diff
@@ -17,8 +17,10 @@
  */
 package uk.ac.ebi.ampt2d.metadata.persistence.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
The newly added imports don't seem to be necessary in this class.

Reply:
Removed all unnecessary imports in this and other entity classes, fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/b12ad94aa306c0e8eb84f967c15b5cb6baa5fc2c